components/intl.rst

```diff
@@ -79,6 +79,20 @@ The ``Languages`` class provides access to the name of all languages::
     $language = Languages::getName('fr');
     // => 'French'
 
+If you want to you can also use the ISO 639-2 three-letter language codes instead of
+the ISO 639-1 two-letter codes. (They are here called alpha3 codes.)
```

Comment:
```
You can also use the ISO 639-2 three-letter language codes instead of
the ISO 639-1 two-letter codes, respectively called alpha3 and alpha2 codes. 
```